app/components/c-collapse.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+
+  <div :class="b({expanded: isExpanded})">
+    <div :class="b('toggle')" @click="setState">
+      <e-icon :class="b('icon')" :inline="true" icon="i-plus"/>
+      {{ title }}
+    </div>
+
+    <div :class="b('content')">
+      <div :class="b('inner')">
+        <!-- @slot Used for item content -->
+        <slot></slot>
+      </div>
+    </div>
+
+  </div>
+
+</template>
+
+<script>
+  import EventBus from '@/setup/event-bus';
+
+  /**
+   * Displays collapsible content panels. Use **c-collapse-group** as a wrapper for multiple items
+   * and if only one item should be open at the same time (:one-active=true).
+   */
+  export default {
+    name: 'c-collapse',
+    // components: {},
+    // mixins: [],
+
+    props: {
+      /**
+       * Sets state of the item
+       */
+      active: {
+        type: Boolean,
+        required: false,
+        default: false,
+      },
+
+      /**
+       * Title of the toggle item
+       */
+      title: {
+        type: String,
+        required: true,
+      },
+    },
+
+    data() {
+      return {
+        isExpanded: this.$props.active
+      };
+    },
+
+    // computed: {},
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    mounted() {
+      /**
+       * Listens if c-collapse-group (parent) has received event and emitted back
+       *
+       * @returns   {object}  child   Previous toggled element
+       */
+
+      EventBus.$on('c-collapse-group.toggle', (payload) => {
+        const toggledElement = payload.toggledCollapse;
+        const toggleGroup = payload.component;
+
+        // close all items except toggled if 'one-active=true'
+        if (payload.component.$el.contains(this.$el) && this !== toggledElement && toggleGroup.$props.oneActive) {
+          this.isExpanded = false;
+        }
+      });
+    },
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    methods: {
+      /**
+       * Sets state and emits event to EventBus
+       */
+      setState() {
```

Comment:
- [ ] I would name this method "toggleState" because you don't set a state, you toggle just the current state